httpobs/website/api_v2.py

```diff
@@ -0,0 +1,134 @@
+import sys
+from datetime import datetime, timedelta
+
+from flask import Blueprint, jsonify, request
+
+import httpobs.database as database
+import httpobs.scanner as scanner
+from httpobs import STATE_FAILED
+from httpobs.conf import API_COOLDOWN, DEVELOPMENT_MODE
+from httpobs.scanner.grader import get_score_description
+from httpobs.website import add_response_headers
+from httpobs.website.utils import valid_hostname
+
+api_v2 = Blueprint("api_v2", __name__)
+
+
+@api_v2.route("/analyze", methods=["GET", "OPTIONS", "POST"])
+@add_response_headers(cors=True)
+def api_post_scan_hostname():
+    status_code = 200
+    scan = {}
+    tests = {}
+
+    host = request.args.get("host", "").lower().strip()
+    try:
+        site_id = database.select_site_id(host, create=False)
+    except IOError:
+        return {
+            "error": "database-down",
+            "text": "Unable to connect to database",
+        }, 500
```

Comment:
I'd suggest 503 to keep 500 for uncaught runtime errors, assuming Flask defaults to 500 if the handler throws an uncaught error.

```suggestion
        }, 503
```